packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -508,7 +508,6 @@ export const announcePreviewService = ({
     clientService,
     userStore,
     authStore,
-    capabilityStore,
```

Comment:
I guess now there's no more need to pass it to the `announcePreviewService` method here.

Reply:
True !

Reply:
True !